### PR TITLE
DOCSP-28548 Update Scheduled Triggers Documentation for Skip Events

### PR DESCRIPTION
**Pull request by lindseymoore**
### Jira

- https://jira.mongodb.org/browse/DOCSP-28548

### Staged Changes

- [Scheduled Triggers](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28548/triggers/scheduled-triggers/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [ ] Create Jira ticket for corresponding docs-realm update(s), if any
- [ ] Checked/updated Admin API
- [ ] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/triggers/scheduled-triggers.txt [hunk 1]
```diff
@@ -113,11 +116,34 @@ Scheduled Triggers have the following configuration options:
...
-   * - | :guilabel:`Trigger Name`
+   * - | :guilabel:`Name`
        | ``name: <string>``
 
      - Required. The name of the Trigger.
 
+   * - | :guilabel:`Enabled`
+       | ``disabled: <boolean>``
+
+     - Enabled by default. Used to enable trigger.
```
Comment (cbullinger) [inline source/triggers/scheduled-triggers.txt:127]:
```suggestion
     - Enabled by default. Used to enable the Trigger. 
```
i'm a little confused by how we're using capital "T" Trigger vs. trigger -- seems inconsistent

File: source/triggers/scheduled-triggers.txt [hunk 2]
```diff
@@ -113,11 +116,34 @@ Scheduled Triggers have the following configuration options:
...
+   * - | :guilabel:`Enabled`
+       | ``disabled: <boolean>``
+
+     - Enabled by default. Used to enable trigger. 
+   
+   * - | :guilabel:`Skip Events on Re-Enable`
+       | ``skip_catchup_event: <boolean>``
+
+     - Disabled by default. When option is enabled, missed trigger events 
+       while trigger is disabled are skipped when trigger is re-enabled.
```
Comment (cbullinger) [inline source/triggers/scheduled-triggers.txt:133]: same as above -- why lowercase "t" trigger throughout this description?

**Review by cbullinger — APPROVED** [review]
A couple nits/questions, but overall lgtm!

File: source/triggers/scheduled-triggers.txt (hunk 2 above)
Comment (lindseymoore) [reply source/triggers/scheduled-triggers.txt:133]: In favor of changing them to lowercase

File: source/triggers/scheduled-triggers.txt [hunk 3]
```diff
@@ -113,11 +116,34 @@ Scheduled Triggers have the following configuration options:
...
        For scheduled Triggers, this value should be set to ``SCHEDULED``.
 
-   * - | :guilabel:`Trigger Name`
+   * - | :guilabel:`Name`
        | ``name: <string>``
 
      - Required. The name of the Trigger.
 
+   * - | :guilabel:`Enabled`
+       | ``disabled: <boolean>``
```
[the remark this answers is not in the document]
Comment (lindseymoore) [reply source/triggers/scheduled-triggers.txt:125]: yes!

**Pull request merged**